File: order-ms/src/main/java/com/nexeum/orderms/dto/request/OrderProductItem.java
package com.nexeum.orderms.dto.request;

import lombok.Data;

import java.math.BigDecimal;

@Data
public class OrderProductItem {
    private Long productId;
    private String productName;
    private Long productQuantity;
    private BigDecimal totalPrice;
    private BigDecimal unitPrice;
    private String productImageUrl;
}

File: order-ms/dto/request/OrderProductItem.cs
namespace order_ms.dto.request
{
    public class OrderProductItem
    {
        public long ProductId { get; set; }
        public string? ProductName { get; set; } = string.Empty;
        public long ProductQuantity { get; set; }
        public decimal TotalPrice { get; set; }
        public decimal PricePerUnit { get; set; }
        public string? ProductImageUrl { get; set; } = string.Empty;
    }
}